src/ts/ucp.ts

```diff
@@ -86,6 +86,15 @@ $(document).ready(function() {
         });
     });
 
+
+    // select the already selected permission for templates
+    $(document).on('click', '.modalToggle', function() {
```

Comment:
what was wrong with the previous approach? (JS-less)

Reply:
The pre-selection of the current permission level was missing. So the modal dialogue opened always with the first option: Public.

Reply:
if you look in src/templates/edit.html, the select approach is JS-less, like it was here also before this change. So I don't understand why it didn't work the same way as it works for experiments/items. https://github.com/elabftw/elabftw/blob/master/src/templates/edit.html#L31-L34

Reply:
The experiment opens up as a separate entity so we can grab the open experiment fields in HTML templates same as like `Entity.entityData.canread` but with Templates they does not open as a separate rather it works on active class, when we click it changes the active class to that template and by this the experiment approach wouldn't work here.

Reply:
I see.